.github/scripts/generate_ci_workflows.py

```diff
@@ -180,7 +193,7 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
     ),
     CIWorkflow(
         arch="windows",
-        build_environment="periodic-pytorch-win-vs2019-cuda11-cudnn8-py3",
+        build_environment="pytorch-win-vs2019-cuda11-cudnn8-py3",
```

Comment:
While we're in here, do we really need to prefix the name with `pytorch-`? It doesn't add much information IMO